sec-priv-q.md

```diff
@@ -0,0 +1,303 @@
+reveal the values they receive to each other and they faithfully execute the
+defined MPC protocol, the value of the match key is never revealed to any
+entity.
+
+IPA defines an MPC protocol that provides sites with an aggregated report that
+can be used for attribution.  The MPC system chosen guarantees *malicious
+security* in the *two out of three honest majority* setting.  In this setting,
+both privacy for inputs and security against tampering are guaranteed as long as
+two of the three helper parties faithfully executes the protocol.  It takes two
+corrupted or malicious helpers that actively work together to access the input
+data provided about users or to spoil the output of the computation.  Any attack
+by a single helper party can only cause the protocol to abort.
+
+User agent implementers, on behalf of their users, need to select helper parties
+that can be trusted to correctly execute the protocol and to only reveal the
+output of any processing they perform.  In addition to the cryptographic
+protections built into the MPC protocol design, the specification recommends
+that helper parties be bound by contractual terms with user agent
+implementations to execute the protocol correctly.
+
+Each top-level browsing context receives a unique encryption and secret-sharing
+of the chosen match key, which ensures that the information visible to the site
+or individual helper parties does not reveal any cross-site correlation between
+users.  A top-level context can make its value available to nested contexts;
+nested contexts do not receive a different value.  This value is retained by the
+user agent in site-specific storage; subsequent requests result in the same
+value being returned.
+
+
+## Key Challenges
+
+There two major components to this work that require special attention from a
+privacy perspective:
+
+1. This proposal uses information - match keys - that could be used to perform
+   cross-site tracking if the protections in the proposal were to fail.  The API
+   allows any web site to request and receive this information from user agents.
+   The proposal includes multiple measures that are designed to protect this
+   information.
+
+2. The aggregated information that is provided to sites is based on the the use
+   of match keys.  The use of differential privacy ensures that there is some
+   protection for the contribution of individual users.  With no limit to the
+   time over which queries can be made, the privacy loss experienced by
+   participating users is similarly unbounded.  The design only limits the rate
+   at which sites gain this information.
+
+Any conclusions about the privacy properties of the API will depend on an
+assessment of the adequacy of the design for both types of protection.
+
+
+## Standard Questions
+
+### What information might this feature expose to Web sites or other parties,
+    and for what purposes is that exposure necessary?
```

Comment:
This part of the question isn't addressed yet. Should be something to the effect of "learning aggregate information about activity that is specifically designed to be cross site, most commonly, paid advertising"